src/transaction.js

```diff
@@ -12,20 +12,11 @@ var zconstants = require('./constants')
 var zaddress = require('./address')
 var zopcodes = require('./opcodes')
 
-/* More info: https://github.com/ZencashOfficial/zen/blob/master/src/script/standard.cpp#L377
```

Comment:
This comment is unrelated to the method itself.

Reply:
Nice!